CreeDictionary/API/search/affix.py

```diff
@@ -125,15 +124,15 @@ def query_would_return_too_many_results(query: InternalForm) -> bool:
     return len(query) <= settings.AFFIX_SEARCH_THRESHOLD
 
 
-def fetch_english_keywords_with_ids():
+def fetch_target_language_keywords_with_ids():
     """
     Return pairs of indexed English keywords with their corresponding Wordform IDs.
     """
     # Slurp up all the results to prevent walking the database multiple times
     return tuple(EnglishKeyword.objects.all().values_list("text", "lemma__id"))
```

Comment:
Should this class also be changed to not reference a specific language?

Reply:
I’m leaving that for when the `EnglishKeyword` database table gets renamed.